antfs-cli.py

```diff
@@ -337,24 +335,24 @@ def main():
         g = AntFSCLI(options.upload)
         try:
             g.start()
-        except:
+        finally:
             g.stop()
-            raise
     except Device.ProfileVersionException as e:
-        print "\nError:", str(e), "\n\nThis means that", \
-                AntFSCLI.PRODUCT_NAME, "found that your data directory " \
-                "stucture was too old or too new. The best option is " \
-                "probably to let", AntFSCLI.PRODUCT_NAME, "recreate your " \
-                "folder by deleting your data folder, after backing it up, " \
-                "and let all your files be redownloaded from your sports " \
-                "watch."
+        print("\nError:", str(e), "\n\nThis means that", \
+              AntFSCLI.PRODUCT_NAME, "found that your data directory " \
+                                     "stucture was too old or too new. The best option is " \
+                                     "probably to let", AntFSCLI.PRODUCT_NAME, "recreate your " \
+                                                                               "folder by deleting your data folder, after backing it up, " \
```

Comment:
Yes. I'll use % formatting to avoid this (my editor is auto-indenting like this, and PEP8 checking is valid, but it would look better with % formatting).